### ISSUE TITLE
SVG looks significantly different to the TTF counterpart

### ISSUE THREAD

**Issue by alexstyl** [google/material-design-icons#1531]
The SVG version of `Pause` looks significantly different to the ttf version.

| TTF | SVG|
| --- | --- | 
| ![image 74](https://github.com/google/material-design-icons/assets/1665273/7a18c152-b8cf-499b-98c3-fe468e0532fc) | <img width="168" alt="Screenshot 2023-06-22 at 01 02 54" src="https://github.com/google/material-design-icons/assets/1665273/c1f0b672-8c3f-4b1c-a3ff-bfe0c9f3165c"> |

As you can see, the thickness and shape of the two icons are different. 

On the left you see 'pause' from MaterialSymbolsRounded with FILL1 wght400 GRAD 0 opsz40.

On the right you see materialsymbolsrounded/pause_fill1_40px.svg.

I believe that the two should look alike. I double checked and it seems like the ttf is the meant respective version of the svg.

**Comment by tphinney** [comment]
@alexstyl Is the SVG perhaps from Material Icons instead of Material Symbols?

**Comment by alexstyl** [comment]
@tphinney I am using the SVGS from `/symbols/web/`. I am assuming those are the Symbols ones, aren't they?

**Comment by Dionis-mgn** [comment]
Another example is `battery_charging_60` (and similar) icon. In repo all the versions of those icons are fillled: [not filled](https://github.com/google/material-design-icons/blob/master/symbols/web/battery_charging_60/materialsymbolsoutlined/battery_charging_60_48px.svg) and [filled](https://github.com/google/material-design-icons/blob/master/symbols/web/battery_charging_60/materialsymbolsoutlined/battery_charging_60_fill1_48px.svg) are the same icons.

fonts.google.com has it right though: [unfilled](https://fonts.gstatic.com/s/i/short-term/release/materialsymbolsoutlined/battery_charging_60/default/48px.svg) and [filled](https://fonts.gstatic.com/s/i/short-term/release/materialsymbolsoutlined/battery_charging_60/fill1/48px.svg)

**Comment by tphinney** [comment]
@alexstyl yes, those are the Symbols ones and should in theory be correct.

In this particular case, I remember well how the filled versions of the Pause icon were updated earlier in the Symbols font this year. It was an unusual case/decision (from a design perspective) and we discussed it in our internal repo WAY more than any sane person would expect. 😝 

@Dionis-mgn Yes, another thing that was enhanced at some later time, although this goes further back.

**Comment by alexstyl** [comment]
thanks for confirming @tphinney. are there any plans to update the svgs in the repo any time soon?

**Comment by sabinebarrera** [comment]
@alexstyl did you choose the right configuration for your SVG symbol?
<img src="https://github.com/google/material-design-icons/assets/1494577/86aac5ad-317b-4714-9d38-f357616edb0d" width="450"/>

When using a ttf symbol you must set your Text's font correctly:
```
Text(
   text = "\ue87d", // Favorite icon
   fontFamily = materialIconsRoundedFontFamily,
   fontSize = 24.sp,
   fontWeight = FontWeight.Light
)
```

**Comment by alexstyl** [comment]
@sabinebarrera I am aware of the configurations of the ttf. You can find the configurations I am using in the 1st post.

The bug discussed is about inconsistencies between the ttf and the svgs founds in this repository. It is not related to Jetpack Compose.

**Comment by tphinney** [comment]
This was fixed back last May.
The SVG currently live on the website matches the TTF version you show on the left. (Your other screenshot seems to be broken.)

I am going to https://github.com/google/material-design-icons/blob/master/symbols/web/pause/materialsymbolsrounded/pause_fill1_40px.svg
(change that to 24 or 20 px to get the other sizes, all are good)

So, your current preview must have something wrong.
